app/uk/gov/hmrc/hec/models/hecTaxCheck/HECTaxCheckMatchRequest.scala

```diff
@@ -14,12 +14,13 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.hec.models
```

Comment:
maybe the stuff to do with matching can live under a separate `matching` package so that `hecTaxCheck` is all about the stored BE data structures

Reply:
That was what I was aiming for - to put all BE models in `hecTaxCheck`, we can move the matching models `HECTaxCheckMatchResult` and `HECTaxCheckMatchRequest` into a separate `matching` package . The other option is to move them closer to where it's being used in `TaxCheckService`. Thoughts?

Reply:
I think it'd be more visible in a separate `matching` package - if it was only used by the `TaxCheckService` then maybe it'd belong there where it's more local but it's used by both the service and the controller

Reply:
updated https://github.com/hmrc/hec/pull/54/commits/aa47933b2d812011e367018d0a077f329c4e5a24

created a separate package `taxCheckMatch` (let me know if you can think of a better name)